### ISSUE TITLE
Perhaps, misspelling in setup.py for long_description

### ISSUE THREAD

**Issue by rojaster** [linkedin/asciietch#4]
```python

description = 'A library for graphing charts using ascii characters.'
try:
    with io.open('README.md', encoding="utf-8") as fh:
            long_description = fh.read()
except IOError:
    long_description = description

#.......

setup(
    name='asciietch',
    version='1.0.2',
    description=description,
    long_description=description, # <----- long_description?

```

**Comment by durgaswaroop** [comment]
This has been fixed in PR #8.

**Comment by scallister** [comment]
You're right, closing this out, ty! :)
